content/expressions/control-structures.md

```diff
@@ -74,7 +74,7 @@ Here it is not obvious whether the `else` is an alternative to the first or the
 
 The big difference for control structures between Pony and other languages is that in Pony everything is an expression. In languages like C++ and Java `if` is a statement, not an expression. This means that you can't have an `if` inside an expression, there has to be a separate conditional operator '?'.
 
-In Pony there are no statements, there are only expressions; everything hands back a value. Your `if` statement hands you back a value. Your `for` loop (which we'll get to a bit later) hands you back a value.
+In Pony control flow statements like this are all expressions that hand back a value. Your `if` statement hands you back a value. Your `for` loop (which we'll get to a bit later) hands you back a value.
```

Comment:
I'd probably include `while` loops in here, if the intent is to list statements which usually aren't expressions in other programming languages.